dataset/all_time.json

```diff
@@ -0,0 +1,87 @@
+[
+   {
+      "rank": 1,
+      "name": "Jianfeng Jiang",
+      "score": 897
+   },
+   {
+      "rank": 2,
+      "name": "Yuke Peng",
+      "score": 582
+   },
+   {
+      "rank": 3,
+      "name": "LI Qing",
+      "score": 358
+   },
+   {
+      "rank": 4,
+      "name": "Junyang Zhang",
+      "score": 301
+   },
+   {
+      "rank": 5,
+      "name": "Tate, Hongliang Tian",
+      "score": 215
+   },
+   {
+      "rank": 6,
+      "name": "Chen Chengjun",
```

Comment:
Cheng Chengjun and cchanging are the same guy. Yuke Peng and sdww0 are the same guy. Could you figure out why the same person has two different names listed in the dataset?

Reply:
I have checked and it's because the author list is incomplete. I have added their information to author list